runtime/tests/interpreter/interpreter_test.go

```diff
@@ -1055,6 +1055,25 @@ func TestInterpretEqualOperator(t *testing.T) {
       fun testUnicodeStrings(): Bool {
           return "caf\u{E9}" == "cafe\u{301}"
       }
+
+	  fun testEqualPaths(): Bool {
+		// different domains
+		return /public/foo == /public/foo &&
+			  /private/bar == /private/bar &&
+			  /storage/baz == /storage/baz
+	  }
+
+	  fun testUnequalPaths(): Bool {
+		return /public/foo == /public/foofoo ||
+			  /private/bar == /private/barbar ||
+			  /storage/baz == /storage/bazbaz
+	  }
+
+	  fun testCastedPaths(): Bool {
+		let foo: StoragePath = /storage/foo
+		let bar: PublicPath = /public/foo
+		return (foo as Path) == (bar as Path)
+	  }
```

Comment:
Mixing tabs and spaces can cause formatting to look weird on GitHub, hence these three functions having different indentation than the others. If you find and replace all tabs with 4 spaces in a file, the Go autoformatter should replace those with tabs everywhere except inside the test strings.